### ISSUE TITLE
Will this sheet be submitted to Roll20 for public use?

### ISSUE THREAD

**Issue by Anduh** [lelajcorson/Aces-and-Eights-Character-Sheet#1]
Are there any reason this sheet haven't been submitted to Roll20's [character sheet repo](https://github.com/Roll20/roll20-character-sheets)?

I can help out with getting it approved to the repo, or just do the small stuff and submit there for you. If the code would be submitted to Roll20's repo, it would fall under MIT licence.

I'm also a bit curios about what these two sections of the code was intending to achieve, as roll20 is meant to block all JavaScript that isn't inside their custom  `<script type="text/worker">`-elements.

```
<script src="http://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.4.4/jquery.min.js" type="text/javascript"></script>
```

**Comment by lelajcorson** [comment]
Thank you for offering! I appreciate someone taking an interest in this :). Honestly, I just haven't gotten around to it and there are still some features that I want to add, but I'm not sure how. For example, I wanted to have a button that showed/hid the different modifiers that made up the main accuracy and speed modifiers, but I couldn't figure out how to do that. I would love to have this added to the Roll20 repo though!

What is the MIT license?

As for those lines of code, I was just playing around with things and trying to get them to work, but they didn't actually do anything. I just forgot to delete them :). In this project, I was teaching myself a lot of the code by looking things up online, as I don't have a ton of HTML and CSS experience. I have definitely gotten better at HTML and CSS, but I'm not super familiar with JS.

**Comment by Anduh** [comment]
> For example, I wanted to have a button that showed/hid the different modifiers that made up the main accuracy and speed modifiers, but I couldn't figure out how to do that.

You could take a look at the Examples from the [CSS Wizardry](https://wiki.roll20.net/CSS_Wizardry) page on how to hide/show or swap the visibility of sections on a Roll20 sheet.

> What is the MIT license?

[MIT License](https://github.com/Roll20/roll20-character-sheets/blob/master/LICENSE): "A short and simple permissive license with conditions only requiring preservation of copyright and license notices. Licensed works, modifications, and larger works may be distributed under different terms and without source code."

>  In this project, I was teaching myself a lot of the code by looking things up online, as I don't have a ton of HTML and CSS experience.

Yeah I've learned most of my HTML/CSS working on various Roll20 sheets. Is this a sheet for the old game, or the more recent edition that was on Kickstarter some years ago?

**Comment by lelajcorson** [comment]
Ok I will look into that! This is for the older version I believe. Is the new version very different?

**Comment by lelajcorson** [comment]
What is the process like for getting it added? I know there are some guidelines on the roll20 wiki for what you have to have. I think I need to add a few things to make it comply with that, but after that how would I go about it?

**Comment by Anduh** [comment]
I have no idea, just asked so the sheet can be labelled correctly when submitted to Roll20.

**Comment by Anduh** [comment]
> What is the process like for getting it added? I know there are some guidelines on the roll20 wiki for what you have to have. I think I need to add a few things to make it comply with that, but after that how would I go about it?

The main thing is to not use `<table>` elements, and I've looked over your sheet, and it passes by flying colors code-wise, one just need to create the [sheet.json ](https://github.com/Roll20/roll20-character-sheets#creating-your-own) and fill it out correctly, while doing the right thing with making a PR from a forked repo. 

If you want to do it yourself, here is a guide to it: https://wiki.roll20.net/Github

Otherwise, I can easily do that for you, as it's a thing I've done around a dozen times by now.

**Comment by lelajcorson** [comment]
Awesome! I made one and I think I filled it out correctly, but let me know if I didn't. I'll also add a preview image. I saw something else on the wiki about having files with Unix endings? Is that something I need to do as well?

**Comment by Anduh** [comment]
Looks solid.

In some text editors you can specify to save the files with unix endings, but I might have seen Roll20 fixing the line-ending thing themselves, so don't think that's a major problem.

In the instruction section you could mention this is a sheet for the game published in (specific year), and maybe say something helpful about using the sheet if there is something that isn't completely intuitive.
Otherwise following the guide I linked should show the right process with forks and pull requests.

**Comment by lelajcorson** [comment]
Will do! Thank you!!

**Comment by Anduh** [comment]
You can ping me in the PR if you want me to give it a look while it's waiting for Roll20 to review it. They do the reviews once a week during Mondays/Tuesdays.

**Comment by lelajcorson** [comment]
Do I need to clone my forked repository to my desktop / is there a way for me to not store all of the files on my computer? I would like to avoid taking up a bunch of storage on my computer if possible :)

**Comment by Anduh** [comment]
Uh, If you can manage to upload the sheet's files directly to your github fork, pretty sure it can be done, but I use my computer as inbetween so don't remember exactly how it's done.

If you go to the [roll20 repo](https://github.com/Roll20/roll20-character-sheets) and press the "fork" button, it will create a personal fork of the repo for you on github, and then you can experiment with adding the sheet to it, from where you then do the Pull Request.

**Comment by lelajcorson** [comment]
ok thanks! I'll experiment
